composer.json

```diff
@@ -42,7 +42,8 @@
         "amphp/parser": "^1",
         "amphp/windows-registry": "^0.3",
         "daverandom/libdns": "^2.0.1",
-        "ext-filter": "*"
+        "ext-filter": "*",
+        "danog/libdns-native": "^0.1.0"
```

Comment:
I don't like introducing a dependency we don't control just for the blocking fallback here. Could you contribute what's necessary to either LibDNS or `amphp/dns`, please?